docs/access_token-pop.md

```diff
@@ -0,0 +1,178 @@
+
+3. **How does Client -> Resource Provider protocol change between Bearer and Pop access tokens**.
+
+   In the bearer flow, when a client requests a resource from an RP, the client provides an **authorization header** containing the AT. 
+
+   In the PoP flow, when a client requests a resource from an RP, the client provides an authorization header containing the Signed Http Request**(SHR)**, which itself contains the AT: 
+
+   |                               | **Bearer**            | **PoP**                |
+   | ----------------------------- | --------------------- | ---------------------- |
+   | **Authentication** **Method** | Bearer Authentication | PoP Authentication     |
+   | **AT issued by STS**          | “Bearer” AT           | “PoP” AT               |
+   | **Authorization Header**      | Bearer AT             | **SHR** **{ PoP AT }** |
+
+4. **Configure MSAL to request Proof-of-Possession Access tokens**
+
+   4.1	For an interactive acquireToken request, create an instance of MSALInteractiveTokenParameters as shown below.
+
+   **MSALInteractiveTokenParameters**
+
+   ```objective-c
+   #if TARGET_OS_IPHONE
+       UIViewController *viewController = ...; // Pass a reference to the view controller that should be used when getting a token interactively
+       MSALWebviewParameters *webParameters = [[MSALWebviewParameters alloc] initWithAuthPresentationViewController:viewController];
+   #else
+       MSALWebviewParameters *webParameters = [MSALWebviewParameters new];
+   #endif 
+   
+   MSALInteractiveTokenParameters *interactiveParams = [[MSALInteractiveTokenParameters alloc] initWithScopes:scopes webviewParameters:webParameters];
+   ```
+
+   For a silent acquireToken request, create an instance of MSALSilentTokenParameters as shown below.
+
+   **MSALSilentTokenParameters**
+
+   ```
+   NSError *error = nil;
+   MSALAccount *account = [application accountForIdentifier:accountIdentifier error:&error];
+   if (!account)
+   {
+       // handle error
+       return;
+   }
+       
+   MSALSilentTokenParameters *silentParams = [[MSALSilentTokenParameters alloc] initWithScopes:scopes account:account];
+   ```
+
+   4.2	MSALTokenParameters which is the parent class for MSALInteractiveTokenParameters and MSALSilentTokenParameters has been extended to include an additional property called authenticationScheme as shown below.
+
+   ```
+   /**
+    Authentication Scheme to access the resource
+    */
+   @property (nonatomic, nullable) id<MSALAuthenticationSchemeProtocol> authenticationScheme;
+   ```
+
+   Create an instance of authenticationScheme which is either MSALAuthenticationSchemeBearer or MSALAuthenticationSchemePop. MSALAuthenticationSchemeBearer is the default authentication scheme used by the library for bearer access tokens and does not require explicit declaration.
+
+   MSALAuthenticationSchemePop has three required parameters, 
```

Comment:
From below looks like it has 2 required parameters

Reply:
Good catch. Addressed.